pkg/util/kube/pod.go

```diff
@@ -37,3 +37,21 @@ func IsPodRunning(pod *corev1api.Pod) error {
 
 	return nil
 }
+
+// IsPodRunning does a well-rounded check to make sure the specified pod has been scheduled into a node and in a stable status.
+// If not, return the error found
+func IsPodScheduled(pod *corev1api.Pod) error {
```

Comment:
Could you extract the dup code of `IsPodRunning` and `IsPodRunningOnNode` into a shared package private func?

Reply:
Done